{{cookiecutter.project_slug}}/modules/manifest.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+from django.urls import path, include
+
+MODULES_PACKAGE_NAME = "modules"
+MODULES_DIR = f"{Path.cwd()}/{MODULES_PACKAGE_NAME}/"
+APPS = Path(MODULES_DIR).rglob("apps.py")
+
+
+def get_modules():
+    try:
+        modules = []
+        python_package_separator = "."
+        for app in APPS:
+            app_name = app.as_posix().replace(MODULES_DIR, f"{MODULES_PACKAGE_NAME}/")
+            app_name = python_package_separator.join(app_name.split("/")[:-1])
+            modules.append(app_name)
+        return modules
+    except (ImportError, IndexError):
+        pass
```

Comment:
Won't this require removing/updating `modules/urls.py` and `modules/admin.py` since loading each as an app makes what they do now unnecessary?

Reply:
Not necessarily, we can keep this feature too, which means that not only do the apps get installed automatically, their urls and admin get included as well.

Reply:
I've updated the urls.py one to catch `ProgrammingError` which will be raised before migrations are ran when drf tries to run the queryset.